README.md

```diff
@@ -1,201 +1,93 @@
   <br/>
-
-  <h3><b>Microverse README Template</b></h3>
-
+  <h3><b>Pennywisepal</b></h3>
 </div>
 
-<!-- TABLE OF CONTENTS -->
-
-# 📗 Table of Contents
+# Table of Contents
 
-- [📖 About the Project](#about-project)
-  - [🛠 Built With](#built-with)
-    - [Tech Stack](#tech-stack)
-    - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
-- [💻 Getting Started](#getting-started)
-  - [Setup](#setup)
+- [About the Project](#about-project)
+  - [Built With](#built-with)
+  - [Live Demo](#live-demo)
+- [Getting Started](#getting-started)
   - [Prerequisites](#prerequisites)
-  - [Install](#install)
+  - [Installation](#installation)
   - [Usage](#usage)
-  - [Run tests](#run-tests)
+  - [Run Tests](#run-tests)
   - [Deployment](#deployment)
-- [👥 Authors](#authors)
-- [🔭 Future Features](#future-features)
-- [🤝 Contributing](#contributing)
-- [⭐️ Show your support](#support)
-- [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
-- [📝 License](#license)
-
-<!-- PROJECT DESCRIPTION -->
-
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
-
-## 🛠 Built With <a name="built-with"></a>
-
-### Tech Stack <a name="tech-stack"></a>
-
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
-
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
-
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
-
-<!-- Features -->
-
-### Key Features <a name="key-features"></a>
-
-> Describe between 1-3 key features of the application.
+- [Authors](#authors)
+- [Future Features](#future-features)
+- [Contributing](#contributing)
+- [Show Your Support](#show-your-support)
+- [Acknowledgements](#acknowledgements)
+- [License](#license)
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+## About the Project <a name="about-project"></a>
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+Pennywisepal is a budgeting app built with Ruby on Rails and PostgreSQL. It helps users track and manage their finances effectively.
 
-<!-- LIVE DEMO -->
+### Built With <a name="built-with"></a>
 
-## 🚀 Live Demo <a name="live-demo"></a>
+The project is built using the following technologies:
 
-> Add a link to your deployed project.
+- Ruby on Rails
+- PostgreSQL
+- HTML
+- CSS
 
-- [Live Demo Link](https://google.com)
+### Live Demo <a name="live-demo"></a>
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+A live demo of Pennywisepal can be accessed [here](https://your-live-demo-link.com).
+A video demo can be found [here](https://www.loom.com/share/921f8efb29fc4c51aeb2fdc56a2e236b)
```

Comment:
- [ ] Kindly add a link to the deployed app, the current link doesn't work. This is one of the requirements that need to be made to guarantee project approval.